src/UXClient/Components/HierarchyNavigation/HierarchyNavigation.scss

```diff
@@ -638,16 +651,31 @@
     .tsi-hierarchyNavigationContextMenu{
```

Comment:
i think we get rid of the blue border on this thing too and just go with the box shadow...the theme blue border looks kind of dated now

Reply:
nick is also sort of in the process of removing it.  we might be able to go with $gray2 or whatever the light blue color is in light theme, but i think we can just make it looking like a white card with box shadow and call it good.  idk let me know what you think, talk it over with azima too maybe